tests/vmi_multus_test.go

```diff
@@ -587,24 +645,15 @@ var _ = Describe("SRIOV", func() {
 		}
 
 		// Create two sriov networks referring to the same resource name
-		result := virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov")).
-			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
-		result = virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov2")).
-			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov2", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
-		result = virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov-link-enabled")).
-			Body([]byte(fmt.Sprintf(sriovLinkEnableConfCRD, "sriov-link-enabled", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
+		ipam := "{ \"type\": \"host-local\", \"subnet\": \"10.1.1.0/24\" }"
+		err := createSriovNetwork("sriov", tests.NamespaceTestDefault, "sriov_net", tests.NamespaceTestDefault, ipam, 0, false, true, false)
+		Expect(err).NotTo(HaveOccurred(), "should create SriovNetwork sriov")
+
+		err = createSriovNetwork("sriov2", "sriov_net", tests.NamespaceTestDefault, tests.NamespaceTestDefault, ipam, 0, false, true, false)
```

Comment:
nit: 
maybe you can use constant bool (`const trusted  = true`). I think it will make it easier to understand what configurations
are given to the SRIOV-network.